### PR TITLE
Improve the transport key print statement to ensure that we don't get…

### PR DESCRIPTION
**Pull request by rhc54**
… zero fields as this can be a problem for PSM

@hppritcha Could you please try this out?

### CODE REVIEW

File: orte/mca/ess/pmi/ess_pmi_module.c [hunk 1]
```diff
@@ -161,17 +161,7 @@ static int rte_init(void)
         error = "getting univ size";
         goto error;
     }
-    orte_process_info.max_procs = u32;
-
```
Comment (jjhursey) [inline orte/mca/ess/pmi/ess_pmi_module.c]: It looks like this is reverting the PMIX_UNIV_SIZE commit from earlier, was that intended?

Comment (rhc54) [inline orte/mca/ess/pmi/ess_pmi_module.c]: crud - no, will fix. Not sure how that happened.

Thx!

**Comment by hppritcha** [comment]
yep, after 2.0.0 dust settles.

**Comment by hppritcha** [comment]
works for me, both 1 process and multi-process on a single node.
:+1:

**Pull request merged**